app/controllers/idv/otp_verification_controller.rb

```diff
@@ -70,6 +71,26 @@ def handle_otp_confirmation_failure
       end
     end
 
+    def save_in_person_notification_phone
+      return unless IdentityConfig.store.in_person_send_proofing_notifications_enabled
+      return unless is_in_person_session?
+      # future tickets will support voice otp
+      return unless idv_session.user_phone_confirmation_session.delivery_method == :sms
+
+      current_enrollment.notification_phone_configuration =
+        NotificationPhoneConfiguration.new(
+          phone: idv_session.user_phone_confirmation_session.phone,
+        )
+    end
+
+    def current_enrollment
```

Comment:
```suggestion
    def establishing_enrollment
```
I would call this `establishing_enrollment`, since thats how we typically refer to enrollments elsewhere (`<status>_enrollment`)